bin/apisix

```diff
@@ -82,7 +87,9 @@ local ngx_tpl = [=[
 # This is a read-only file, do not try to modify it.
 
 master_process on;
-
+{% if is_root_path then %}
+user root;
```

Comment:
Using `root` as the user is dangerous and is likely to be exploited by hackers

Reply:
Oh okays, but it won't start when in root dir rite?, I guess from the previous message I understood to allow the user to start but only print warnings?

Reply:
yes, you are right:)
IMO, We can print warnings, but we don't need to deal with the permission problem, and we don't need to be compatible with the situation of using root to start Apache APISIX.

Reply:
okay, then ill remove the root user configs, and only settle to printing warnings.